bluesky_queueserver/server/server.py

```diff
@@ -151,7 +160,25 @@ async def _pop_from_queue_handler():
     """
     Pop the last item from back of the queue
     """
-    msg = await re_server._send_command(command="pop_from_queue")
+    msg = await re_server.send_command(command="pop_from_queue")
+    return msg
+
+
+@app.get("/get_history")
+async def _get_history_handler():
+    """
+    Returns the plan history (list of dict).
```

Comment:
```suggestion
    Returns the plan history (list of dicts).
```
... or "list of dictionaries".